docker-compose.yml

```diff
@@ -1,18 +1,27 @@
----
-version: "3.2"
+version: "3.7"
+
+volumes:
+  state:
 
 services:
-  dev:
-    container_name: creditcoin-node
-    image: paritytech/ci-linux:c4f0daf3-20220113
-    working_dir: /var/www/creditcoin-node
-    ports:
-      - "9944:9944"
-    environment:
-      - CARGO_HOME=/var/www/creditcoin-node/.cargo
+  creditcoin:
+    container_name: creditcoin-validator
     volumes:
-      - .:/var/www/creditcoin-node
-      - type: bind
-        source: ./.local
-        target: /root/.local
-    command: bash -c "cargo build --release && ./target/release/creditcoin-node --dev --ws-external"
+      - state:/creditcoin-node/data
+    image: gluwa/creditcoin
+    command:
+      - --name
+      - atodorov validator
+      - --validator
+      - --bootnodes
+      - "/dns4/testnet-bootnode.creditcoin.network/tcp/30333/p2p/12D3KooWG3eEuYxo37LvU1g6SSESu4i9TQ8FrZmJcjvdys7eA3cH"
```

Comment:
I think this needs to be parameterized.
I guess the test network is a good default, but if the network is killed and restarted, or boot nodes change, etc. etc. it's _yet another place_ we'll need to remember to change infra details.

Reply:
>  it's yet another place we'll need to remember to change infra details.

that also holds true if you employ parameters.

Reply:
Yeah, I was thinking that as I was writing it.
Will CI at least fail if the boot node isn't accessible?

Reply:
because of PoS you might be able to query that there is no next era planned since no staking has occurred on the network, if you use the RPC of the node being started, but you'll have to wait for it to sync?

Reply:
So this could become a bogus compose file in a couple of months too.
I'll note this and we can address it later in the week.

Reply:
>  Will CI at least fail if the boot node isn't accessible?

Apparently not, howeverthis is due to the fact that the boot node is made available in the chain spec.

```diff
       - validator
       - --validator
       - --bootnodes
-      - "/dns4/testnet-bootnode.creditcoin.network/tcp/30333/p2p/12D3KooWG3eEuYxo37LvU1g6SSESu4i9TQ8FrZmJcjvdys7eA3cH"
+      - "/dns4/bogus-boot-node.example.com/tcp/30333/p2p/12D3KooWG3eEuYxo37LvU1g6SSESu4i9TQ8FrZmJcjvdys7eA3cH"
       - --chain
       - test
```

Logs:
```
creditcoin-validator | 2023-06-20 19:57:13 📦 Highest known block at #67525    
creditcoin-validator | 2023-06-20 19:57:13 〽️ Prometheus exporter started at 127.0.0.1:9615    
creditcoin-validator | 2023-06-20 19:57:13 Running JSON-RPC HTTP server: addr=127.0.0.1:9933, allowed origins=["http://localhost:*", "http://127.0.0.1:*", "https://localhost:*", "https://127.0.0.1:*", "https://polkadot.js.org"]    
creditcoin-validator | 2023-06-20 19:57:13 Running JSON-RPC WS server: addr=127.0.0.1:9944, allowed origins=["http://localhost:*", "http://127.0.0.1:*", "https://localhost:*", "https://127.0.0.1:*", "https://polkadot.js.org"]    
creditcoin-validator | 2023-06-20 19:57:16 👶 Starting BABE Authorship worker    
creditcoin-validator | 2023-06-20 19:57:16 🔍 Discovered new external address for our node: /ip4/217.18.251.214/tcp/30333/p2p/12D3KooWF2R2YLC3ogzUNvoxQY26F9BwztyLShCResvYtGZVQoBg    
creditcoin-validator | 2023-06-20 19:57:17 [67852] 💨 unstaked , outcome: Ok(())    
creditcoin-validator | 2023-06-20 19:57:17 [67872] 💨 unstaked , outcome: Ok(())    
creditcoin-validator | 2023-06-20 19:57:18 ⚙️  Syncing, target=#68837 (3 peers), best: #67921 (0x2d86…1fbc), finalized #67584 (0xa1c1…8fd5), ⬇ 44.9kiB/s ⬆ 3.1kiB/s    
```

Reply:
Wait, can we just remove the boot node from the docker-compose all together then?